WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -52,6 +52,15 @@ final class ProductSelectorViewModel: ObservableObject {
     ///
     @Published private(set) var productRows: [ProductRowViewModel] = []
 
+    /// Determines if multiple item selection is supported, `false` by default.
+    /// Mainly used to allow multi-selection on the Order Creation flow
```

Comment:
Not a blocker, but this comment feels to specific for a generic class like this one

Reply:
Ah, yes! Removed on https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/03b231d9239b592c5846d442c2a9e3495b4532c0